src/olympia/devhub/views.py

```diff
@@ -1333,7 +1334,8 @@ def _submit_upload(request, addon, channel, next_details, next_finish,
         check_validation_override(request, form, addon, version)
         addon_update = {}
```

Comment:
we just use this to avoid multiple updates to Addon, but now `admin_review` isn't a property then the only possible update is `status = amo.STATUS_NOMINATED` so we can do it straight away.

Reply:
Good point, doing that.